urth-viz-explorer.html

```diff
@@ -134,10 +134,20 @@
                 width: 200px;
             }
 
+            paper-dropdown-menu.option-choice {
+                display: inline-block;
+            }
+
             paper-card-collapse {
                 display: block;
             }
 
+            .clear-option-btn {
+                color: #9b9b9b;
+                width: 2.5em;
+                height: 2.5em;
```

Comment:
There is a bit much padding on the delete button that causes the row height to be a bit higher. You can see the effect if there are 2 optional inputs in a row. It is simple to fix, just use similar styling to how I did the sort direction button, but looks like we need slightly different values. Try this here:

```css
    padding: 0;
    width: 1.3em;
    height: 1.3em;
```